src/lib/parts.ts

```diff
@@ -430,14 +456,32 @@ export class EventPart implements Part {
     if (this._pendingValue === noChange) {
       return;
     }
-    if ((this._pendingValue == null) !== (this.value == null)) {
-      if (this._pendingValue == null) {
-        this.element.removeEventListener(this.eventName, this);
-      } else {
-        this.element.addEventListener(this.eventName, this);
+
+    const newListener = this._pendingValue;
+    const oldListener = this.value;
+    const shouldRemoveListener = newListener == null ||
+        oldListener != null &&
+            (newListener.capture !== oldListener.capture ||
+             newListener.once !== oldListener.once ||
+             newListener.passive !== oldListener.passive);
+    const shouldAddListener =
+        newListener != null && (oldListener == null || shouldRemoveListener);
+
+    if (shouldRemoveListener) {
+      try {
+        // This can throw if the options don't match the options that the
+        // listener was added with. This should never happen.
```

Comment:
If it should never happen, I wouldn't try/catch; it would be a bug that must be fixed either way, better to let the user know it's a library bug (throw) rather than log ("maybe I did something wrong").

Reply:
Er, or are you protecting against new options being added?  That might be a good reason to be a little more forgiving...

Reply:
Oh, but you use a whitelist of properties to copy, so even if the user added new ones, the options passed to aEL would be the same, so I guess not?

Reply:
No, this was just a try/catch because removeEventListener can throw if the arguments don't match a previous add call. There's a small chance that a user could change the options on a reference to the old listener and cause an exception because the options are pull off as needed. I could save to ole options to prevent that possibility.

Reply:
I changed to pull off the options so they can't be changed out from under the part.

To address the other part of your comment: this isn't to protect against new options. Ideally we'd like to let those pass through, there's just some difficulties with function objects that require a little bit more code. ie, spreading a method into an object was throwing for me because that was accessing `callee` which throws (I guess it's enumerable, but throws in strict mode? ugh). So we just need to filter some things out. I didn't want to deal with it honestly 🤷‍♂️ I can file an issue...

Reply:
Yeah, we also want the diff (do the old options match the new ones) to determine when to remove to be fast, and supporting an arbitrary list makes that harder (slower).  So... probably ok as is; we can probably rev `lit-html` faster than browsers can ship a new event option. :P